packages/apidom-reference/test/dereference/strategies/asyncapi-3/server-bindings-object/index.ts

```diff
@@ -0,0 +1,33 @@
+import path from 'node:path';
+import { assert } from 'chai';
+import { toValue } from '@swagger-api/apidom-core';
+import { mediaTypes } from '@swagger-api/apidom-ns-asyncapi-3';
+import { fileURLToPath } from 'node:url';
+
+import { loadJsonFile } from '../../../../helpers.ts';
+import { dereference } from '../../../../../src/index.ts';
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+const rootFixturePath = path.join(__dirname, 'fixtures');
+
+describe('dereference', function () {
+  context('strategies', function () {
+    context('asyncapi-3', function () {
+      context('Server Bindings Object', function () {
+        context('given in components/securitySchemes field', function () {
```

Comment:
```suggestion
        context('given in server/bindings field', function () {
```

And we also need to add the test for `components/serverBindings`